static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js

```diff
@@ -57,30 +59,118 @@ const getIndexSiteId = (): string => {
     return site ? site.id : '';
 };
 
+const contains = (sizes: PrebidSize[], size: PrebidSize): boolean =>
+    Boolean(sizes.find(s => s[0] === size[0] && s[1] === size[1]));
```

Comment:
How can you be sure that the array is not empty and has two or more elements?

Reply:
If it's empty the result should be false.
And a `PrebidSize` is defined as `[number, number]` so should always have exactly two elements.